src/senaite/microorganism/content/microorganism.py

```diff
@@ -18,15 +18,179 @@
+from AccessControl import ClassSecurityInfo
+from plone.autoform import directives
+from plone.supermodel import model
 from senaite.core.catalog import SETUP_CATALOG
+from senaite.core.content.base import Container
+from senaite.core.schema import UIDReferenceField
+from senaite.core.z3cform.widgets.uidreference import UIDReferenceWidgetFactory
+from senaite.microorganism import messageFactory as _
 from senaite.microorganism.interfaces import IMicroorganism
+from zope import schema
 from zope.interface import implementer
+from Products.CMFCore import permissions
 
 
-@implementer(IMicroorganism)
-class Microorganism(Item):
+class IMicroorganismSchema(model.Schema):
+    """Schema interface
+    """
+
+    title = schema.TextLine(
+        title=u"Title",
+        required=True,
+    )
+
+    description = schema.Text(
+        title=u"Description",
+        required=False,
+    )
+
+    gram_stain = schema.Choice(
+        title=_(u"Gram stain"),
+        vocabulary="senaite.microorganism.vocabularies.gram_stains",
+        required=False,
+    )
+
+    shape = schema.Choice(
+        title=_(u"Shape"),
+        vocabulary="senaite.microorganism.vocabularies.shapes",
+        required=False,
+    )
+
+    glass = schema.Bool(
+        title=_(u"GLASS organism"),
+        description=_(
+            "Whether this is an infectious organism included in the Global "
+            "Antimicrobial Resistance Surveillance System (GLASS) for the "
+            "the collection, analysis and sharing of Antimicrobial Resistance "
+            "(AMR) data at a global level"
+        ),
+        required=False,
+    )
+
+    multi_resistant = schema.Bool(
+        title=_(u"MRO Organism"),
+        description=_(
+            "Whether this organism is considered multi resistant (MRO)",
+        ),
+        required=False,
+    )
+
+    mro_phenotype = schema.TextLine(
+        title=_(u"MRO phenotype"),
+        required=False,
+    )
+
+    category = UIDReferenceField(
+        title=_(u"Category / Group"),
+        description=_(
+            u"The category or group this microorganism belongs to"
+        ),
+        allowed_types=("MicroorganismCategory", ),
+        multi_valued=False,
+        required=False,
+    )
+
+    directives.widget(
+        "category",
+        UIDReferenceWidgetFactory,
+        catalog=SETUP_CATALOG,
+        query={
+            "portal_type": "MicroorganismCategory",
+            "is_active": True,
+            "sort_on": "title",
+            "sort_order": "ascending",
+        },
+        display_template="<a href='${url}'>${title}</a>",
+        columns=[
+            {
+                "name": "title",
+                "width": "30",
+                "align": "left",
+                "label": _(u"Title"),
+            }, {
+                "name": "description",
+                "width": "70",
+                "align": "left",
+                "label": _(u"Description"),
+            },
```

Comment:
Better use `Title` and `Description` instead, because this will use the metadata captured from the `Title()` and `Description()` getters.
However, it will be actually treated the same in the future:
https://github.com/senaite/senaite.core/pull/2295/files#diff-dfea22b9534555110627324ead3c52fe94f710363d41a38bf81161035476ca6cR57